script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -96,22 +96,22 @@ As if the world had turned into a frying pan.
 [sha:440d2cf8581090653fcc957790caa86cc64a2f85]{
 -- Page 4, Offset 21311. Mods: @k, @e.
 -- えーん　えんえん
-Uwah... waah...
+%{i}Uwah... waah...%{/i}
 }
 [sha:440d2cf8581090653fcc957790caa86cc64a2f85]{
 -- Page 4, Offset 21312. Mods: @k, @e.
 -- えーん　えんえん
-Uwah... waah...
+%{i}Uwah... waah...%{/i}
 }
 [sha:45274b29b87bbaecf500085b6780109974324567]{
 -- Page 4, Offset 21313. Mods: @k, @e.
 -- えーん　えんえ
-Uwah... waah.
+%{i}Uwah... waah...%{/i}
 }
-[sha:26beb972a16ee66dcd57191b91bbd5515ebf9fd0]{
+[offset:21314]{
 -- Page 4, Offset 21314. Glued. Mods: @x, @k, @e.
 -- ん
-..
+%{nothing}
```

Comment:
how does this render ig?

Reply:
It's a glued empty string, so you don't really see it.